cluster/ubuntu/util.sh

```diff
@@ -508,6 +505,12 @@ function provision-masterandnode() {
       cp ~/kube/minion/* /opt/bin/
 
       service etcd start
+      service kube-apiserver start
```

Comment:
We make it clear in document about this. I had no experience with ubuntu 15.10 though.